diff --git a/node_modules/frappe-ui/src/charts/measureText.ts b/node_modules/frappe-ui/src/charts/measureText.ts
--- a/node_modules/frappe-ui/src/charts/measureText.ts
+++ b/node_modules/frappe-ui/src/charts/measureText.ts
@@ -19 +19 @@
-export const CHART_FONT_FAMILY = 'InterVar, Inter, sans-serif'
+export const CHART_FONT_FAMILY = 'Vazirmatn, InterVar, Inter, sans-serif'
diff --git a/node_modules/frappe-ui/src/components/Sidebar/SidebarCollapseToggle.vue b/node_modules/frappe-ui/src/components/Sidebar/SidebarCollapseToggle.vue
--- a/node_modules/frappe-ui/src/components/Sidebar/SidebarCollapseToggle.vue
+++ b/node_modules/frappe-ui/src/components/Sidebar/SidebarCollapseToggle.vue
@@ -2 +2 @@
-  <SidebarItem :label="isCollapsed ? 'Expand' : 'Collapse'" @click="toggle">
+  <SidebarItem :label="isCollapsed ? __('Expand') : __('Collapse')" @click="toggle">
diff --git a/node_modules/frappe-ui/experimental/ListView/ListFooter.vue b/node_modules/frappe-ui/experimental/ListView/ListFooter.vue
--- a/node_modules/frappe-ui/experimental/ListView/ListFooter.vue
+++ b/node_modules/frappe-ui/experimental/ListView/ListFooter.vue
@@ -12,13 +12,11 @@
           <Button
             v-if="showLoadMore"
-            label="Load More"
+            :label="__('Load More')"
             @click="emit('loadMore')"
           />
           <div v-if="showLoadMore" class="mx-3 h-[80%] border-l" />
           <div class="flex items-center gap-1 text-base text-ink-gray-5">
-            <div>{{ options.rowCount || '0' }}</div>
-            <div>of</div>
-            <div>{{ options.totalCount || '0' }}</div>
+            <div>{{ __('{0} of {1}', [options.rowCount || '0', options.totalCount || '0']) }}</div>
           </div>
         </div>
       </slot>
